profile.md

```diff
@@ -1,4 +1,4 @@
-# Open Service Broker API Profile
+# Open Service Broker API Profile v2.12
```

Comment:
hmm, should this say "(master - might contain changes that are not yet released)" to align with master?  If so, we should probably have a PR for the v2.12 branch that says "v2.12".